config.yaml

```diff
@@ -13,4 +13,18 @@ services:
     conditions:
       - "[STATUS] == 200"
       - "[BODY].deleted == false"
-      - "len([BODY].text) > 0"
\ No newline at end of file
+      - "len([BODY].text) > 0"
+
+auto-discover-k8s-services: true
+k8s-service-suffix: ".svc.environment.local"
+
+k8s-service-config:
+  interval: 30s
+  conditions:
+    - "[STATUS] == 200"
+
+exclude-suffix:
+  - primary
+  - canary
+
+k8s-cluster-mode: "out"
```

Comment:
Could you move these parameters under `kubernetes`?

i.e.

```yaml
kubernetes:
  cluster-mode: "out"
  auto-discover: true
  exclude-suffix:
    - primary
    - canary
  service-template:
    interval: 30s
    conditions:
      - "[STATUS] == 200"
```

Reply:
Sure will make this change.

Reply:
@TwinProduction Have made the changes.